Oglle/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+	"author": {
+		"name": "Oglle",
+		"id": "641614088215986186"
+	},
+	"service": "OGLLE.COM.BR",
+	"description": {
+        "pt-BR": "Precisa de um BOT para Discord? Edição/criação de sites? Ou qualquer outro serviço fora do comum? Este é o site feito para você!",
+        "en": "Need a BOT for Discord? Website editing/creation? Or any other out-of-the-ordinary service? This is the site made for you!"
+	},
+	"url": ["oglle.com.br/#Inicio"],
+	"version": "0.0.1",

```

Comment:
URL doesnt need to be in an array and should just be oglle.com.br
Also version 1.0